lua/rustaceanvim/neotest/init.lua

```diff
@@ -41,10 +41,35 @@ NeotestAdapter.root = function(file_name)
   return cargo.get_root_dir(file_name)
 end
 
+---@param name string Name of directory
+---@param rel_path string Path to directory, relative to root
+---@param root string Root directory of project
+---@return boolean
+---@diagnostic disable-next-line: unused-local
+NeotestAdapter.filter_dir = function(name, rel_path, root)
+  return rel_path ~= 'target'
+end
```

Comment:
```suggestion
---@param _ string Name of directory
---@param rel_path string Path to directory, relative to root
---@param _ string Root directory of project
---@return boolean
---@diagnostic disable-next-line: unused-local
NeotestAdapter.filter_dir = function(_, rel_path, _)
  return rel_path ~= 'target'
end
```

Reply:
Trying out this change locally, this causes a `duplicate-doc-param` warning. I'm still pretty new to Lua, but is there a better way to have multiple unused arguments than disabling that `duplicate-doc-param` warning for those two lines?

Reply:
Ah, sorry. You can just delete the two `--@param` annotations.

Reply:
I've gone ahead and simply commented out those two param annotations instead, just so the types and meanings of the ignored parameters are known should the `filter_dir` logic ever need to be updated.